### DISCUSSION TITLE
For this integration exactly WHO has access to my email password?

### DISCUSSION THREAD

**Discussion by KruseLuds** [moralmunky/Home-Assistant-Mail-And-Packages#729] [Q&A] [answered]
For this integration exactly WHO has access to my email password?

**Comment by firstof9** [comment, marked as answer]
You

**Reply by KruseLuds** [reply]
Sorry I meant for this integration - anyone besides me?

**Reply by moralmunky** [reply]
All of the settings, connections, and processing are done inside of your own local instance of Home Assistant. The only connections it makes outside of the local system are to the mail server you provide.
